Automattic/jetpack
# Protect: Use js-packages/components IconTooltip

## Description
Replace custom `IconTooltip` implementation with js-packages version

### Other information:


## Jetpack product discussion


- https://github.com/Automattic/jetpack-scan-team/issues/1476

## Does this pull request change what data or activity we track or use?


- No

## Testing instructions:






* Checkout this branch and refer to views in the screenshots below
   * FirewallHeader pre-upgrade and post-downgrade with outdated automatic rules enabled
   *  PaidAccordion when a fixer is stale (to be removed shortly)
* Ensure the popover handles and displays correctly
* Ensure no regressions are introduced

## Screenshots

FirewallHeader:
![Screen Shot on 2024-10-22 at 11-23-49](https://github.com/user-attachments/assets/81881248-5ec5-4351-8f91-4be068a8d891)

PaidAccordion:
![Screen Shot on 2024-10-22 at 11-14-49](https://github.com/user-attachments/assets/6301baee-515e-4d90-871d-62d801c0769a)
